lib/venice/receipt.rb

```diff
@@ -4,6 +4,11 @@ module Venice
   class Receipt
     MAX_RE_VERIFY_COUNT = 3
 
+    module EnvName
```

Comment:
Why did we move this module to `Receipt`? Can we extract `Environment` module into the root scope? It's still used in multiple places.